src/backend/adapters/angular2.ts

```diff
@@ -316,24 +320,58 @@ export class Angular2Adapter extends BaseAdapter {
   _getComponentInput(debugEl: any): Object[] {
     let inputs = [];
     if (debugEl.componentInstance) {
+      // Get inputs from @Component({ inputs: [] })
       const metadata = Reflect.getOwnMetadata
         ('annotations', debugEl.componentInstance.constructor);
 
-      inputs = (metadata && metadata.length > 0) ?
-        metadata[0].inputs : [];
+      inputs = (metadata && metadata.length > 0 && metadata[0].inputs) || [];
+
+      // Get inputs from @Input()
+      const propMetadata = Reflect.getOwnMetadata('propMetadata',
+        debugEl.componentInstance.constructor);
+
+      if (propMetadata) {
+        for (const key of Object.keys(propMetadata)) {
+          for (const meta of propMetadata[key]) {
+            if (meta.constructor.name === (InputMetadata as any).name) {
+              if (inputs.indexOf(key) < 0) { // avoid duplicates
```

Comment:
@sumitarora I am assuming you get that error at runtime? Also, when you get the error the application is not loaded, or is it more of a warning?